.env.example

```
@@ -1,5 +1,14 @@
+TZ=UTC
 PORT=3333
-HOST=0.0.0.0
+HOST=localhost
+LOG_LEVEL=info
+APP_KEY=
 NODE_ENV=development
-APP_KEY=73rHnSEgVPLfFLpTk8omwzBrMk-c75rX
-DRIVE_DISK=local
+DB_HOST=127.0.0.1
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DB_HOST key should go before the HOST key

(UnorderedKey)